openedx/core/djangoapps/credentials/tasks/v1/tasks.py

```diff
@@ -142,25 +154,46 @@ def handle_notify_credentials(options, course_keys):
             notify_programs=options['notify_programs']
         )
 
-    logger.info('notify_credentials finished')
```

Comment:
I removed this log line purposefully as I didn't think it was particularly useful.